konduit-serving-pipeline/src/test/java/ai/konduit/serving/pipeline/impl/testpipelines/callback/CallbackStep.java

```diff
@@ -35,6 +32,7 @@
         PipelineRegistry.registerStepRunnerFactory(new CallbackPipelineFactory());
     }
 
+    @Schema(description = "A function which takes in argument and produces a result")
```

Comment:
All pipeline steps in src/test/java should be marked as ignored in the test - not annotated.